### PR TITLE
Update the sentence capitalization for v5.22 \b{wb}

### LINKED ISSUE #100: perlfaq4: =head2 How do I capitalize all the words on one line?
This section uses a quote from Dr. Strangelove, and then slips into the Flintstones and it doesn't make sense to me.   Also, it is out-of-date.  \b{wb} is better than plain \b for finding word breaks and in fact treats an embedded apostrophe as not being a break.  This could be a teaching moment about these newer capabilities of Perl

### PR DESCRIPTION
**Pull request by briandfoy**
To fix #100.

### CODE REVIEW

**Review by davorg — DISMISSED** [review]

**Comment by briandfoy** [comment]
What needs to happen for this to move forward?

**Comment by briandfoy** [comment]
This PR has been approved. What needs to happen for it to be merged?

**Comment by davorg** [comment]
I'd rather that someone other than me merge it (given that I was the person who approved it).

**Comment by briandfoy** [comment]
Here's the patch file in case this PR loses the diff when I clean up my fork later this month.

[0001-Update-the-sentence-capitalization-for-v5.22-b-wb-10.patch](https://github.com/perl-doc-cats/perlfaq/files/12862151/0001-Update-the-sentence-capitalization-for-v5.22-b-wb-10.patch)

File: lib/perlfaq4.pod [hunk 1]
```diff
@@ -882,39 +882,57 @@ X<Text::Autoformat> X<capitalize> X<case, title> X<case, sentence>
...
-                 (^\w)    #at the beginning of the line
-                   |      # or
-                 (\s\w)   #preceded by whitespace
-                   )
-                /\U$1/xg;
+	my $string = "A camel's journey";
+	$string =~ s/\b(\w)/\U$1/g;  # A Camel'S Journey
+
+How is Perl supposed to know not to capitalize that C<s>? You could work
+harder to look for preceeding whitespace or the beginning of the string:
```
Comment (karenetheridge) [inline lib/perlfaq4.pod:913]:
```suggestion
harder to look for preceding whitespace or the beginning of the string:
```

**Pull request merged**